external/sketch/ansible/roles/proxypass-setup/README.md

```diff
@@ -0,0 +1,54 @@
+proxypass-setup
+=========
+
+Optional role.
+
+This will automatically setup the Proxy Protocol chain from all edge nodes to middle. And middle to proxy
+
+Best used for when using only one middle sketch.
+
+This assumes you are pointing middle to proxy01.
+
+Requirements
+------------
+
+Must be ran after initial Sketch setup as nginx is required to be installed and loaded. Make sure you copy below into your playbook
+
+If using OCI replace `REPLACE ME` with the following:
+
+"jq '.resources[] | select(.type == \"oci_core_instance\" and .name == \"proxy\") | .instances[0].attributes.public_ip' ../../oci/terraform.tfstate"
+
+If using AWS replace `REPLACE ME` with the following:
+
+"jq '.resources[] | select(.type == \"aws_instance\" and .name == \"proxy\") | .instances[0].attributes.public_ip' ../../aws/terraform.tfstate"
+
+Example Playbook
+----------------
+
+Add to the end of sketch-playbook.yml 
+
+```
+- name: Configure Host Groups and Variables
```

Comment:
Instead of adding something to `sketch-playbook.yml` create a role that can be applied to sketch